### PR TITLE
Update GH Page to reflect new Java version Recommendation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds a bullet to "News and Events" section of codice.org/ddf advising users to upgrade to 8u60 or higher.
#### Who is reviewing it?

@vinamartin @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR.

@shaundmorris
@stustison

#### How should this be tested?

open `index.html` in browser.

#### What are the relevant tickets?

[DDF-2365](https://codice.atlassian.net/browse/DDF-2365)
#### Screenshots (if appropriate)
n/a
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,6 +96,11 @@ <h3 class="panel-title">
                         </div>
                         <div class="list-group scrollable-list">
                              <a class="list-group-item">
+                                <h4 class="list-group-item-heading">Friday, August 5, 2016</h4>
+                                <p class="list-group-item-text">The DDF Development Team has updated the recommended system configuration for DDF. Please ensure a Java version of  or later to take advantage of performance enhancements.
```
Comment (vinamartin) [inline index.html]: Maybe this is a silly question, but how does this display the version? I'm seeing "    ".

Comment (ricklarsen) [inline index.html]: Nope. Legit question. Fixed.

Comment (vinamartin) [inline index.html]: Awesome! Thought I was missing something!

**Comment by vinamartin** [comment]
LGTM 👍

**Comment by jrnorth** [comment]
LGTM

**Comment by Lambeaux** [comment]
:shipit:

**Pull request merged**